POSserver/server/schemas/note.py

```diff
@@ -1,8 +1,17 @@
-# from graphene import relay, List, ObjectType
+from graphene import relay, List, ObjectType
 from graphene_django import DjangoObjectType
 from server.models import Note
 
 
 class Note_Type(DjangoObjectType):
     class Meta:
         model = Note
+        filter_fields = ["title", "content", "created_at", "modified_at"]
+        interfaces = (relay.Node,)
+
+
+class Query(ObjectType):
+    all_notes = List(Note_Type)
+
+    def resolve_all_notes(self, info, **kwargs):
+        return Note.objects.all()
```

Comment:
Hmm..If you're returning all notes..I'm curious why you need **kwargs in the parameters?